packages/backend/src/server/api/stream/channels/main.ts

```diff
@@ -13,6 +14,8 @@ export default class extends Channel {
 		this.subscriber.on(`mainStream:${this.user!.id}`, async data => {
 			switch (data.type) {
 				case 'notification': {
+					// Ignore notifications from instances the user has muted
+					if (isInstanceMuted(data.body, new Set<string>(this.userProfile?.mutedInstances ?? []))) return;
```

Comment:
```suggestion
					if (data.body.note && isInstanceMuted(data.body.note, new Set<string>(this.userProfile?.mutedInstances ?? []))) return;
```

Reply:
I did a different change to this noticing there is a `.user` on `Packed<'Notification'>` so that any notification from users from muted instances would be muted, not only notes